.buildkite/test-pipeline.yaml

```diff
@@ -44,7 +44,7 @@ steps:
   fast_check: true
   commands:
   # This flashinfer installation will fail on AMD ROCm, so it is set as optional.
-  - pip install https://github.com/flashinfer-ai/flashinfer/releases/download/v0.0.8/flashinfer-0.0.8+cu121torch2.3-cp310-cp310-linux_x86_64.whl || true
+  - pip install https://github.com/flashinfer-ai/flashinfer/releases/download/v0.1.2/flashinfer-0.1.2+cu121torch2.4-cp310-cp310-linux_x86_64.whl || true
```

Comment:
I think that flashinfer does not provide wheels for 3.12 (see `cp310` part of the release name)

https://github.com/flashinfer-ai/flashinfer/blob/main/.github/workflows/release_wheel.yml#L28